src/lib/api/patrons/patron.js

```diff
@@ -15,6 +15,8 @@ const get = async patronPid => {
 };
 
 const list = async queryText => {
+  // NOTE: the '*' in the end of the query will not work for every field
+  // try https://localhost:5000/api/patrons/?q=id:1*
```

Comment:
I will elaborate a bit more because I removed it in first place and other things broke.

I believe it has to do with type of the field we are searching on. For the special case we use the `id` the `https://localhost:5000/api/patrons/?q=id:1*` query will return zero results, while `https://localhost:5000/api/patrons/?q=id:1` will return the patron with `id` 1.

`https://localhost:5000/api/patrons/?q=email:patr` will return nothing while https://localhost:5000/api/patrons/?q=email:patr*` will return the partial matches.

For the time being if you check the `defaultConfig` I am using the `pid` and not the `id` field, so everything works, just wanted to point out my findings while working with `patronApi.list`

Reply:
I have added an optional param wildcard, to disable the `*` on demand.